pywbem/cim_operations.py

```diff
@@ -1255,14 +1298,386 @@ class are to be included in the returned instances, as follows:
+              causes the server-implemented default to be used. :term:`DSP0200`
+              defines that the server-implemented default is `True`.
+
+            This parameter has been deprecated in :term:`DSP0200` and should be
+            set to `False` by the caller.
+
+          DeepInheritance (:class:`py:bool`):
+            Indicates that properties added by subclasses of the specified
+            class are to be included in the returned instances, as follows:
+
+            * If `False`, properties added by subclasses are not included.
+            * If `True`, properties added by subclasses are included.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default to be used. :term:`DSP0200`
+              defines that the server-implemented default is `True`.
+
+            Note, the semantics of the `DeepInheritance` parameter in
+            :meth:`~pywbem.WBEMConnection.EnumerateClasses` and
+            :meth:`~pywbem.WBEMConnection.EnumerateClassNames`
+            is different.
+
+          IncludeQualifiers (:class:`py:bool`):
+            Indicates that qualifiers are to be included in the returned
+            instance, as follows:
+
+            * If `False`, qualifiers are not included.
+            * If `True`, qualifiers are included if the WBEM server implements
+              support for this parameter.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default to be used. :term:`DSP0200`
+              defines that the server-implemented default is `False`.
+
+            This parameter has been deprecated in :term:`DSP0200`. Clients
+            cannot rely on it being implemented by WBEM servers.
+
+          IncludeClassOrigin (:class:`py:bool`):
+            Indicates that class origin information is to be included on each
+            property in the returned instances, as follows:
+
+            * If `False`, class origin information is not included.
+            * If `True`, class origin information is included.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default to be used. :term:`DSP0200`
+              defines that the server-implemented default is `False`.
+
+          PropertyList (:term:`py:iterable` of :term:`string`):
+            An iterable specifying the names of the properties to be
+            included in the returned instances, in any lexical case.
+
+            An empty iterable indicates to include no properties.
+
+            If `None`, all properties are included.
+
+          FilterQueryLanguage (:term:`string`):
+            Optional: a String defining the name of the query language
+            used for the Query argument. This argument must exist if the
+            filterQuery argument is provided.
```

Comment:
This is a named arg with a defaullt value of `None`. So it always exists from the perspective of the called method, and it does not matter whether it was provided by the caller (it could have been provided as `None`).

Therefore, let's use `None` instead of "exists" or "provided", and let's omit "Optional:".